src/Ledger/GovernanceActions.lagda

```diff
@@ -3,11 +3,11 @@
 We introduce three distinct bodies that have specific functions in the new governance framework:
 \begin{enumerate}
 \item
-  a constitutional committee  (henceforth called \AgdaInductiveConstructor{CC})
+  a constitutional committee  (henceforth called \CC)
 \item
-  a group of delegate representatives (henceforth called \AgdaInductiveConstructor{DReps})
+  a group of delegate representatives (henceforth called \DReps)
 \item
-  the stake pool operators (henceforth called \AgdaInductiveConstructor{SPOs})
+  the stake pool operators (henceforth called \SPOs)
```

Comment:
These lines demonstrate the main point of this PR (i.e., to replace the `\Agda*{*}` typesetting macros with our own macros for the Agda tokens that appear in our documentation, so we can be sure they will be typeset consistently and correctly throughout).